docs/volume-scheduling.md

```diff
@@ -92,3 +95,32 @@ spec:
       storage: 8Mi
 EOF
 ```
+
+## Unique allocation of drives for volumes
+
+By default, DirectPV allocates drives for volumes based on the free capacity present on the drives. So, the drive with most free capacity gets selected for a volume provisioning request. For setups that require unique drive allocation for its volumes where one drive cannot share more than one volume, use the below steps to enable unique allocation of drives.
+
+* Create new storage class with `directpv.min.io/volume-claim-id: <any-uid>` using [create-storage-class.sh script](../tools/create-storage-class.sh). Below is an example:
+```sh
+# NOTE: The allocation id must be 47 characters or less and should be valid label value.
+#
+# Create new storage class 'directpv-optimal' with the label 'directpv.min.io/volume-claim-id: 1234'
+$ create-storage-class.sh directpv-optimal 'directpv.min.io/volume-claim-id: 1234'
+```
+
+* Use newly created storage class in [volume provisioning](./volume-provisioning.md). Below is an example:
+```sh
+$ kubectl apply -f - <<EOF
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: sleep-pvc
+spec:
+  volumeMode: Filesystem
+  storageClassName: directpv-optimal
+  accessModes: [ "ReadWriteOnce" ]
+  resources:
+    requests:
+      storage: 8Mi
+EOF
+```
```

Comment:
1. We would need to document this feature and limitation with minio server example
2. The title can be `### Unique drive selection`
3. The section can be under `## Customizing drive selection`